.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+# Changes
+- Change summary/description
+- Any @mention 's
+
+# Related Tickets
```

Comment:
It's not too big of a deal, but I guess Issues would be clearer, up to you if you want to change @danielchen-pyc